helix-term/src/commands.rs

```diff
@@ -1327,7 +1327,7 @@ pub fn scroll(cx: &mut Context, offset: usize, direction: Direction) {
     let text = doc.text().slice(..);
 
     let cursor = coords_at_pos(text, range.cursor(text));
-    let doc_last_line = doc.text().len_lines().saturating_sub(1);
+    let doc_last_line = doc.text().len_lines() - 1;
```

Comment:
On the other hand this might actually crash now since `len_lines()` can return 0

Reply:
`len_lines()` returns `self.root.line_break_count() + 1` and subtracting 1 can't overflow the `usize`.